Cargo.toml

```diff
@@ -5,3 +5,7 @@ exclude = ["contracts"]
 # Resolver has to be set explicitely in workspaces
 # due to https://github.com/rust-lang/cargo/issues/9956
 resolver = "2"
+
+[workspace.dependencies]
+schemars = "0.8.4"
```

Comment:
This is the lowest version that supports `SingleOrVec::contains` which we use in our codebase